### PR TITLE
[2.16.x] DDF-5132 Replace logout UI component with single logout page

### PR DESCRIPTION
**Pull request by SmithJosh**
#### What does this PR do?
Previously the UI used a JS component which handled logging the user
out. The "sign out" buttons throughout the UI called out to it to
handle logouts. This worked for frontend code, but there was no
way for the backend to use it.

This commit modifies the logout page to perform the same functions
as the JS component. Now, both the frontend and backend can use it
by redirecting the user to this page.

In particular, this fixes the session management service's
invalidate() method.

#### Who is reviewing it? 
@brjeter @blen-desta @bakejeyner @stustison
#### Select relevant component teams: 
@codice/security 

#### How should this be tested?
Verify that the following all redirect to the logout page and that the user cannot access the UI again without authenticating once redirected
* The "sign out" buttons in both the Admin UI and Intrigue
* The session inactivity timeout in both the Admin UI and Intrigue

Also, verify that the logout page displays a message when the user signed in via Basic or PKI auth indicating the browser should be closed.

#### Any background context you want to provide?
See the description

#### What are the relevant tickets?
Fixes: #5132

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/security-servlet-logout/src/main/webapp/js/main.js [hunk 1]
```diff
@@ -17,10 +14,68 @@
...
+      $('#close-browser-msg').show()
+      return
+    }
+  }
+
+  const queryParams = parseQueryParams()
+  if (queryParams && queryParams['service']) {
+    $('#link').attr('href', queryParams['service'])
+  } else {
+    $('#link').attr('href', window.location.href.replace(/logout\/.*/, ''))
```
Comment (brjeter) [inline ui/packages/security-servlet-logout/src/main/webapp/js/main.js]: ❓ Doesn't this actually modify the `window.location.href` property? Are we wanting to do that?

Comment (SmithJosh) [reply ui/packages/security-servlet-logout/src/main/webapp/js/main.js]: `window.location.replace` would, but this is just calling the String replace() method

**Comment by brjeter** [comment]
build now

**Comment by SmithJosh** [comment]
Fixed formatting.
build now

**Comment by brjeter** [comment]
build now

**Comment by brjeter** [comment]
Hero successful ✅

**Pull request merged**